src/editor/components/list/list-item.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+interface ListItemProps {
+  children: any
+}
```

Comment:
I think it's overkill to have this here but we can leave it ^^

Reply:
I'd leave it so that if we ever want to better define how "children" should look like it will be easy to do so.